config/default.yml

```diff
@@ -111,6 +111,12 @@ PreCommit:
       - 'config/settings.yml'
       - 'config/settings/**/*.yml'
 
+  Checkstyle:
+    description: 'Analyzing with checkstyle'
+    required_executable: 'checkstyle'
+    flags: ['-c', '/sun_checks.xml']
```

Comment:
Weird indeed, but that's the standard config. The leading slash just indicates that it's built-in, I assume.